components/normalizer/src/lib.rs

```diff
@@ -1134,6 +1134,7 @@ where
                 return Some(starter);
             }
             // Now we need to check if composition with an upcoming starter is possible.
+            #[allow(clippy::unwrap_used)]
```

Comment:
Note: Generally it would be better to put the clippy suppression down closer to where the `unwrap` occurs (where there is already a comment explaining why the `unwrap` is safe), but sometimes you can't apply these to statements, so it's fine.